satellite/analytics/hubspot.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package analytics
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/spacemonkeygo/monkit/v3"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/common/sync2"
+)
+
+var mon = monkit.Package()
+
+// HubSpotConfig is a configuration struct for Concurrent Sending of Events.
+type HubSpotConfig struct {
+	APIKey          string `help:"hubspot api key" default:""`
+	ChannelSize     int    `help:"the number of events that can be in the queue before dropping" default:"10000"`
```

Comment:
Is `10000` reasonable value here?

Reply:
Changed values of ChannelSize to 1000 and ConcurrentSends to 4 as we have 5k - 8k events happening per day.